### ISSUE TITLE
[MAS1.4.11] [Visual Requirements-Live Chat] Luminosity Ratio is less than 3:1 for border outline with respective background colour for “Send button and File upload button “

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1757]
**Actual Result:**
Luminosity Ratio is less than 3:1 for border outline with respective background colour for “Send button and File upload button, Conversation update event detected ”

**Expected Result:**
Luminosity Ratio should be greater than or equal to 3:1 for border outline with respective background colour for “Send button and File upload button ,Conversation update event detected ”
**Note:** This issue exits for throughout application for all button controls 

**Repro Steps:**
1.Open the Bot Framework V4 Emulator.
2.Welcome tab screen gets opened
3. Navigate to the Open a bot and select it 
4.Open bot dialog gets opened and connect the pen bot 
5.Live chat screen gets opened 
6.Navigate to the “Send button and File upload button, Conversation update event detected”
7. Open 'Colour Contrast Analyser' application.
8.Observe the issue

**User Impact:**
Users with low vision or colour blindness needs that difference between contrast of text is always high.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator
Tool: Color contrast Analyzer

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B6CA3BF14-2635-434A-832C-28DD9A18B9FC%7D&file=MAS%201.4.11%20%E2%80%93%20Non-text%20Contrast.docx&action=default&mobileredirect=true&cid=c3837dea-3bb1-4c5a-9374-19ec384f14c6](url)
![MAS1 4 11_Live chat](https://user-images.githubusercontent.com/53518994/63421730-b53d3700-c426-11e9-9abe-6d02f4d8c208.png)

**Comment by Kumar2608** [comment]
Verified in latest bot emulator Version 4.5.2-78641 and issue got fixed Hence closing this bug
Now Luminosity Ratio is 3:1 for  “Send button and File upload button, Conversation update event detected”.
